docs/architecture/adr-020-protobuf-transaction-encoding.md

```diff
@@ -167,36 +196,27 @@ message SignDoc {
 
 In order to sign in the default mode, clients take the following steps:
 
-1. Encode `SignDoc`. (The only requirement of the underlying protobuf
-implementation is that fields are serialized in order).
-2. Sign the encoded `SignDoc` bytes
-3. Build and broadcast `Tx`. (The underlying protobuf implementation must encode
-`TxBody` and `AuthInfo` with the same binary representation as encoded in
-`SignDoc`. If this is a problem for clients, the "raw" types described under
-verification can be used for signing as well.)
+1. Serialize `TxBody` and `AuthInfo` using any valid protobuf serialization
```

Comment:
```suggestion
1. Serialize `TxBody` and `AuthInfo` using any valid protobuf implementation.
```

Reply:
Hah, good one. I actually wanted to say "using any valid protobuf serialization" as in output of a serializer, but this does not work well with the beginning of the sentence. Will take your version.